lib/Ray/include/Color.hpp

```diff
@@ -0,0 +1,76 @@
+/*
+** EPITECH PROJECT, 2021
+** Bomberman
+** File description:
+** Color
+*/
+
+#ifndef COLOR_HPP_
+#define COLOR_HPP_
+
+#include <raylib.h>
+
+namespace Ray {
+    class Color {
+        public:
+            //! @brief A Color constructor
+            //! @param r R-component of color
+            //! @param g B-component of color
+            //! @param g G-component of color
+            //! @param a A-component of color
+            Color(unsigned char r, unsigned char g, unsigned char b, unsigned char a);
+
+            //! @brief A constructor using ray's own color struct
+            //! @info Useful when want to use pre-defined color
+            Color(const ::Color &);
+
+            //! @brief A default copy constructor
+            Color(const Color &) = default;
+
+            //! @brief Create a color from an hexadecimal value
+            Color(int hexValue);
+
+            //! @brief A default destructor
+            ~Color() = default;
+
+            //! @brief An color is assignable
+            Color &operator=(const Color &) = default;
+            
+
+            //! @brief set R-component of color
+            Color &setR(unsigned char r);
```

Comment:
Why do you return a ref to you ?
To  chain calls ?

Reply:
Yes